website/project/model.py

```diff
@@ -1574,7 +1574,7 @@ def remove_node(self, auth, date=None):
         for addon in self.get_addons():
             message = addon.after_delete(self, auth.user)
             if message:
-                status.push_status_message(message, 'info')
+                status.push_status_message(message, kind='info', trust=False)
```

Comment:
> Thank you for using keyword instead of positional arguments.

This is my recommendation going forward. :) Also: I recommend that all future uses of `push_status_message` explicitly specify the `trust=` parameter as a (recommended not required) convention.

This is because we are planning to change the detault to `trust=False` once all pages are converted, and don't want people to be surprised by a behavior change. As long as they specify `trust` explicitly, their pages will continue to work the same way.